akd/Cargo.toml

```diff
@@ -14,9 +14,9 @@ bench = []
 
 [dependencies]
 bincode = "1"
-winter-crypto = "0.1"
-winter-utils = "0.1"
-winter-math = "0.1"
+winter-crypto = "0.2.0"
```

Comment:
One small thing, can you change these to just be "0.2". Specifying the _full_ version doesn't allow us to get minor updates without manually patching. 0.0.X build changes are almost universally safe to upgrade and just contain small bug fixes, so ideally we would like the latest minor revision.

Reply:
Additionally this in all cargo files.